virt-v2v/cold/entrypoint

```diff
@@ -71,6 +71,14 @@ for disk in /dev/block[0-9]* ; do
 	ln -s "$disk" "$DIR/$V2V_vmName-sd$(gen_name "$((num+1))")"
 done
 
+# Adds LUKS keys, if existing.
+if [ -d "/etc/luks" ]; then
+    IFS=$'\n'
+    for line in $(cat /etc/luks/luks); do
+        args+=(--key "$line")
```

Comment:
as I commented at the top-level, I think we can wait to get the --all option because otherwise, users would have to specify the name or the UIID of the device that may be different for different VMs in the plan (we can find the device UID probably, not sure about OVAs though, but as the --all option is just around the corner...)

Reply:
since a selector of all devices will probably not be available for forklift 2.6, we decided not to wait for it
but I talked with @dankenigsberg, raised a vaild point that users that provide such input may expect us to support key per-devices also when that selector would be available.
@liranr23 can we save the user from specifying the device IDs, as we're supposed to have them in the inventory, and ask only for the encryption keys from users?